tern/classes/oci_image.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017-2020 VMware, Inc. All Rights Reserved.
```

Comment:
Super small nit but the copyright should only be  (c) 2020 since this is a new file originating in 2020 :)